docs/release_notes/8.0.4-2.md

```diff
@@ -26,7 +26,7 @@ Percona Server for MongoDB Docker image is now based on Universal Base Image (UB
 
 ### Amazon Linux 2023 support
 
-Percona Server for MongoDB is now available and fully supported on Amazon Linux 2023 (AL23), simplifying its AWS deployment. You can safely run Percona Server for MongoDB on AL23 to build a secure, stable, high-performance environment for developing and running cloud applications, with seamless integration with various AWS services and development tools. You can download tarballs from our [Percona Software Downloads](https://www.percona.com/downloads) page. Navigate to the Percona Server for MongoDB page, select the latest version, and `Generic` Platform. 
+Percona Server for MongoDB is now available and fully supported on Amazon Linux 2023 (AL23), simplifying its AWS deployment. You can safely run Percona Server for MongoDB on AL23 to build a secure, stable, high-performance environment for developing and running cloud applications, with seamless integration with various AWS services and development tools. You can download tarballs from our [Percona Software Downloads](https://www.percona.com/downloads) page. Navigate to the Percona Server for MongoDB page, select the latest version, and `AMAZON LINUX 2023` Platform. 
```

Comment:
I guess it's related to tarballs - those are indeed under 'Generic' section (as well as tarballs for all other OS).
Under 'Amazon Linux 2023' you can find rpm for this specific OS